proto/tendermint/abci/types.proto

```diff
@@ -137,6 +137,9 @@ message RequestPrepareProposal {
   repeated bytes block_data = 1;
   // If an application decides to populate block_data with extra information, they can not exceed this value.
   int64 block_data_size = 2;
+  // votes includes the previous blocks' validator vote in order to reflect the vote data(including vote extension)
+  // in the app_data field of the header.
```

Comment:
```suggestion
  // votes includes all votes from the previous block. This contains vote extension data that can be used in proposal
  // preparation. The votes here will then form the last commit that gets sent in the proposed block.
```
Does this make more sense? I'm trying to understand this use case better as well as improve the way it is communicated